general/update_fars/03_process_fars.py

```diff
@@ -0,0 +1,69 @@
+import pandas as pd
+import os
+import psycopg2
+from dotenv import load_dotenv
+
+# get environment variables
+load_dotenv("rds_conn_vars.env")
+
+# db connection
+TDG_HOST = os.environ["TDG_HOST"]
+TDG_DB_NAME = os.environ["TDG_DB_NAME"]
+TDG_USER_NAME = os.environ["TDG_USER_NAME"]
+TDG_PASSWORD = os.environ["TDG_PASSWORD"]
+
+connection_string = f"host={TDG_HOST} dbname={TDG_DB_NAME} user={TDG_USER_NAME} password={TDG_PASSWORD}"
+conn = psycopg2.connect(connection_string)
+cur = conn.cursor()
+
+for (start_yr, end_yr) in [(2015, 2019), (2017, 2021)]:
+    print(f"Processing {start_yr}_{end_yr} data")
+    query = f"""
+            DROP TABLE IF EXISTS automated.fars_processed_{start_yr}_{end_yr};
+            CREATE TABLE automated.fars_processed_{start_yr}_{end_yr} AS (
+                SELECT
+                    a.st_case,
+                    -- just go through person types in order of vulnerability
+                    CASE
+                        WHEN per_typ && ARRAY[5, 8] THEN 'pedestrian'
+                        WHEN per_typ && ARRAY[6, 7] THEN 'bicyclist'
+                        WHEN per_typ && ARRAY[1, 2, 3] THEN 'motor vehicle'
```

Comment:
Might want to update this to be `ARRAY[1, 2, 3, 9]` per https://crashstats.nhtsa.dot.gov/Api/Public/ViewPublication/813417 (pg. 245)
> In the Person data file, motor vehicle occupants are PER_TYPE = 1, 2, 3, 9.